controllers/upgrade_handlers.go

```diff
@@ -428,7 +435,7 @@ func (u *UpgHandler) PostPivot(ctx context.Context, ibu *ibuv1.ImageBasedUpgrade
 
 	if err := u.RebootClient.DisableInitMonitor(); err != nil {
 		// Don't fail the upgrade on failure here, just log it
-		u.Log.Error(err, "unable to disable LCA init monitor")
+		u.Log.Error(err, "Unable to disable LCA init monitor")
```

Comment:
i think all errors are recommended to start with lowercase letter?

Reply:
This would be an error log, not an error that can be wrapped